bin/pr-review

```diff
@@ -1,5 +1,26 @@
 #!/bin/bash
 
+usage="
+$0: Creates standalone container for reviewing a PR.
```

Comment:
containers (plural) 

I think this should say more, but we should figure that out when we whatever isn't working in this script.